composer.json

```diff
@@ -85,7 +86,145 @@
       "bash tools/scripts/composer/phpword-jquery.sh"
     ]
   },
+  "repositories": {
+    "extrafiles": {
+      "type": "vcs",
+      "url": "https://github.com/totten/ComposerExtraFiles.git"
+    }
```

Comment:
This part is temporary (for demonstration purposes). It needs to be squashed out once we have a more permanent publication of the revised plugin.

Reply:
@seamuslee001 I've updated this to use a published plugin, so it no longer needs the `repositories: ... type: vcs ...` clause.